menas/src/main/scala/za/co/absa/enceladus/menas/services/RunService.scala

```diff
@@ -38,7 +38,7 @@ class RunService @Autowired()(runMongoRepository: RunMongoRepository)
   import scala.concurrent.ExecutionContext.Implicits.global
   import za.co.absa.enceladus.menas.models.Validation._
 
-  @Value("${za.co.absa.enceladus.spline.urlTemplate}")
+  @Value("${spline.urlTemplate}")
```

Comment:
Can you keep this one, please. To avoid (hopefully) conflict with my changes. I will be removing this setting completely in Spline integration PRs.

Reply:
Shouldn't be afraid of merge conflicts. Just rebase your branch.